lib/temporal/configuration.rb

```diff
@@ -3,7 +3,7 @@
 
 module Temporal
   class Configuration
-    attr_reader :timeouts
+    attr_reader :timeouts, :error_handlers
```

Comment:
Let's add a method for adding these handlers rather than expose the whole array. Something like:

```ruby
def on_error(&block)
  @error_handlers << block
end
```